assets/js/theme/product/reviews.js

```diff
@@ -12,38 +12,39 @@ export default class {
         });
 
         this.$reviewsContent = $('#product-reviews');
+        this.$reviewsContentList = $('#productReviews-content', this.$reviewsContent);
         this.$collapsible = $('[data-collapsible]', this.$reviewsContent);
 
         this.initLinkBind();
         this.injectPaginationLink();
-        this.collapseReviews();
+        this.setupReviews();
     }
 
     /**
      * On initial page load, the user clicks on "(12 Reviews)" link
      * The browser jumps to the review page and should expand the reviews section
      */
     initLinkBind() {
-        const $content = $('#productReviews-content', this.$reviewsContent);
-
-        $('#productReview_link').on('click', () => {
-            $('.productView-reviewTabLink').trigger('click');
-            if (!$content.hasClass('is-open')) {
-                this.$collapsible.trigger(CollapsibleEvents.click);
-            }
-        });
+        $('#productReview_link').on('click', () => this.showReviews());
     }
 
-    collapseReviews() {
+    setupReviews() {
         // We're in paginating state, do not collapse
         if (window.location.hash && window.location.hash.indexOf('#product-reviews') === 0) {
-            return;
+            return this.showReviews();
         }
 
         // force collapse on page load
         this.$collapsible.trigger(CollapsibleEvents.click);
     }
 
+    showReviews() {
```

Comment:
Looks like toggleReviews describes this function in a better way

Reply:
but this function can only show reviews, not hide them

Reply:
`expandReviews()` ?..

Reply:
&#9989;